### ISSUE TITLE
Docker entrypoint error

### ISSUE THREAD

**Issue by MariaMayang** [Terrastories/offline-field-kit#2]
`tileserver-1  | GET /health 200 2 - 0.335 ms
db-1          | ls: cannot open directory '/docker-entrypoint-initdb.d/': Permission denied
`
I cloned the repository, executed bash/install, and tried offline terrastories instance

**Comment by rudokemper** [comment]
Operating system: Manjaro Linux (Arch)

**Comment by lauramosher** [comment]
@MariaMayang Can you please provide the full output when running the instance? There is not enough information to determine what went wrong for you.

**Comment by luandro** [comment]
All other containers seemed to be running well, but the tileserver was throwing this error.

**Comment by lauramosher** [comment]
Okay, so can you provide the output of running the tileserver container?

```
docker compose up tileserver
```

**Comment by MariaMayang** [comment]
[mayang@mayang ~]$ docker compose up tileserver
[+] Running 1/1
 ✔ Container mayang-tileserver-1  Creat...                                 0.6s 
Attaching to tileserver-1
tileserver-1  | Starting tileserver-gl v4.7.0
tileserver-1  | Using specified config file from config.json
tileserver-1  | Starting server
tileserver-1  | Listening at http://[::]:8080/
tileserver-1  | Startup complete
tileserver-1  | GET /health 200 2 - 3.926 ms
tileserver-1  | GET /health 200 2 - 1.552 ms
tileserver-1  | GET /health 200 2 - 0.909 ms

**Comment by MariaMayang** [comment]
:)

**Comment by lauramosher** [comment]
Okay, that looks like it is correctly running.

Can you view the Tileserver page if you go to localhost:8080?

**Comment by MariaMayang** [comment]
nothing is showing, do I need to type ant other command "before docker compose up tileserver"  to try?

**Comment by lauramosher** [comment]
Does `terrastories.local:8080` work? And as you are accessing that data, can you see any logs output from the running container?

**Comment by MariaMayang** [comment]
It says it can´t reach the site. I am using firefox which is the main browser in my computer.

**Comment by MariaMayang** [comment]
[mayang@mayang ~]$ docker compose up tileserver
[+] Running 1/0
 ✔ Container mayang-tileserver-1  Creat...                                 0.0s 
Attaching to tileserver-1
tileserver-1  | Starting tileserver-gl v4.7.0
tileserver-1  | Using specified config file from config.json
tileserver-1  | Starting server
tileserver-1  | Listening at http://[::]:8080/
tileserver-1  | Startup complete
tileserver-1  | GET /health 200 2 - 4.320 ms
tileserver-1  | GET /health 200 2 - 0.889 ms
tileserver-1  | GET /health 200 2 - 0.725 ms
tileserver-1  | GET /health 200 2 - 0.662 ms
tileserver-1  | GET /health 200 2 - 0.707 ms
tileserver-1  | GET /health 200 2 - 0.540 ms
tileserver-1  | GET /health 200 2 - 1.328 ms
tileserver-1  | GET /health 200 2 - 0.575 ms
tileserver-1  | GET /health 200 2 - 1.000 ms
tileserver-1  | GET /health 200 2 - 0.984 ms
tileserver-1  | GET /health 200 2 - 0.868 ms
tileserver-1  | GET /health 200 2 - 0.637 ms
tileserver-1  | GET /health 200 2 - 0.789 ms
tileserver-1  | GET /health 200 2 - 0.721 ms
tileserver-1  | GET /health 200 2 - 0.892 ms
tileserver-1  | GET /health 200 2 - 0.787 ms
tileserver-1  | GET /health 200 2 - 0.727 ms
tileserver-1  | GET /health 200 2 - 0.802 ms
tileserver-1  | GET /health 200 2 - 0.667 ms
tileserver-1  | GET /health 200 2 - 0.629 ms
tileserver-1  | GET /health 200 2 - 0.732 ms
tileserver-1  | GET /health 200 2 - 0.613 ms
tileserver-1  | GET /health 200 2 - 0.707 ms
tileserver-1  | GET /health 200 2 - 0.605 ms
tileserver-1  | GET /health 200 2 - 0.586 ms
tileserver-1  | GET /health 200 2 - 0.393 ms
tileserver-1  | GET /health 200 2 - 0.757 ms


tileserver-1  | GET /health 200 2 - 0.694 ms
tileserver-1  | GET /health 200 2 - 1.322 ms

**Comment by lauramosher** [comment]
What is the output of: `docker ps -a | grep tileserver`

**Comment by MariaMayang** [comment]
Laura sorry, I need to step out of the computer for today but will try first thing tomorrow and share back. and big thanks

**Comment by MariaMayang** [comment]
@lauramosher  

19138bff9dc6   terrastories/terrastories-map:latest   "/usr/src/app/docker…"   22 hours ago   Up 2 minutes (healthy)     0.0.0.0:8080->8080/tcp, :::8080->8080/tcp   mayang-tileserver-1
2bf159919945   local-tileserver                       "/usr/src/app/docker…"   6 days ago     Up 2 minutes (unhealthy)   0.0.0.0:8085->8080/tcp, :::8085->8080/tcp   local-tileserver-1

(tileserver words are in red)

**Comment by MariaMayang** [comment]
Hi @lauramosher!  we made it and got Terrastories offline running :) etc host log did not work, so we edited the command and changed from terrastories.local to localhost and it worked.

**Comment by rudokemper** [comment]
That's great @MariaMayang! It's a bit hard to follow what happened, but let us know if you encounter an issue in the future.
